src/components/Teams/TeamMembersPopup.jsx

```diff
@@ -1,27 +1,44 @@
 import React, { useState, useEffect, useCallback } from 'react';
-import { Button, Modal, ModalHeader, ModalBody, ModalFooter, Container, Alert } from 'reactstrap';
+import {
+  Button,
+  Modal,
+  ModalHeader,
+  ModalBody,
+  ModalFooter,
+  Container,
+  Alert,
+  TabContent,
+} from 'reactstrap';
 import MembersAutoComplete from './MembersAutoComplete';
+// import { SwitchComponent } from '@syncfusion/ej2-react-buttons';
+
 import hasPermission from 'utils/permissions';
 import { boxStyle } from 'styles';
 import moment from 'moment';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import { faSort, faSortUp, faSortDown } from '@fortawesome/free-solid-svg-icons';
 import { connect } from 'react-redux';
+import ToggleSwitch from '../UserProfile/UserProfileEdit/ToggleSwitch';
+import InfoModal from './InfoModal';
 
 export const TeamMembersPopup = React.memo(props => {
   const closePopup = () => {
     props.onClose();
-    setSortOrder(0)
+    setSortOrder(0);
```

Comment:
`setSortOrder(0)` is being used before it is defined so I would recommend moving the `closePopup()` function below line 35 after the declared states. 

You typically want to declared all the variables and then the functions of the component.